src/zjs_ocf_client.c

```diff
@@ -930,19 +919,17 @@ static jerry_value_t ocf_get_platform_info(const jerry_value_t function_val,
     // args: device ide
     ZJS_VALIDATE_ARGS(Z_STRING);
     struct ocf_handler *h;
-    jerry_value_t promise = jerry_create_object();
 
     ZJS_GET_STRING(argv[0], device_id, OCF_MAX_DEVICE_ID_LEN + 1);
 
     struct client_resource *resource = find_resource_by_id(device_id);
     if (!resource) {
         ERR_PRINT("resource was not found: %s\n", device_id);
```

Comment:
If you reuse "resource was not found" from below and use template "%s: %s", it will only save that string once.